### PR TITLE
Exclude errors

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/764)

### CODE REVIEW

File: analyzer/src/funTest/assets/projects/synthetic/excludes-with-remove-expected-result.yml [hunk 1]
```diff
@@ -42,6 +42,18 @@ repository:
         - name: "testRuntimeClasspath"
```
Comment (sschuberth) [inline analyzer/src/funTest/assets/projects/synthetic/excludes-with-remove-expected-result.yml]: Commit message nit: Maybe comma after "set"?

File: analyzer/src/funTest/assets/projects/synthetic/excludes-expected-result.yml [hunk 2]
```diff
@@ -143,6 +143,7 @@ analyzer:
             message: "Unresolved: ModuleVersionNotFoundException: Could not find org.example:nonexistent-package:1.0.\n\
```
Comment (sschuberth) [inline analyzer/src/funTest/assets/projects/synthetic/excludes-expected-result.yml]: Commit message nit: Maybe comma after "file"?

File: utils-test/src/main/kotlin/Utils.kt [hunk 3]
```diff
@@ -48,3 +48,4 @@ fun patchExpectedResult(result: File, custom: Pair<String, String>? = null, defi
```
Comment (sschuberth) [inline utils-test/src/main/kotlin/Utils.kt]: Commit message nit: Verb missing after "Maven".

File: model/src/main/kotlin/Error.kt [hunk 4]
```diff
@@ -20,10 +20,15 @@
 package com.here.ort.model
```
Comment (sschuberth) [inline model/src/main/kotlin/Error.kt]: Commit message nit: "converts". Also, the commit message should probably briefly say *why* we do this: "... this is to normalize error messages across platforms to easily compare them to expected results..." or so.

File: utils/src/main/kotlin/Utils.kt [hunk 5]
```diff
@@ -413,6 +413,11 @@ fun String.isValidUrl() =
             false
         }
 
+/**
+ * Replace "\r\n" and "\r" line breaks with "\n".
+ */
+fun String.normalizeLineBreaks() = this.replace("\r\n", "\n").replace("\r", "\n")
```
Comment (sschuberth) [inline utils/src/main/kotlin/Utils.kt]: Maybe use a single Regex like `replaceAll("\\r\\n?", "\n")` instead?

Comment (mnonnenmacher) [reply utils/src/main/kotlin/Utils.kt]: Done.

File: model/src/main/kotlin/Error.kt (hunk 4 above)
Comment (mnonnenmacher) [reply model/src/main/kotlin/Error.kt]: Done.

**Pull request merged**